examples/api/runner.cpp

```diff
@@ -529,3 +538,12 @@ int EXAMPLES_CDECL main(int argc, char** argv) {
 
     return runner.run(); // Return directly from forked processes.
 }
+#if defined(_MSC_VER)
```

Comment:
Suggest copying in rationale:
```c++
// On Windows, print to stderr rather than a popup dialog box (local execution) or completely-absent program termination message (in CI).
```